pilot/.env.example

```diff
@@ -1,13 +1,24 @@
-#OPENAI or AZURE
-ENDPOINT=OPENAI
+# OPENAI or AZURE or OPENROUTER
+ENDPOINT=OPENROUTER
```

Comment:
let's try to keep all defaults same as before. So in this case ENDPOINT=OPENAI

Reply:
Done